ads/vendors/adfox.js

```diff
@@ -13,65 +13,38 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 import {loadScript, validateData} from '#3p/3p';
-import {yandex} from './yandex';
 
 /**
  * @param {!Window} global
  * @param {!Object} data
  */
 export function adfox(global, data) {
   validateData(data, ['adfoxParams', 'ownerId']);
-  loadScript(global, 'https://yastatic.net/pcode/adfox/loader.js', () =>
-    initAdFox(global, data)
+  loadScript(global, 'https://yandex.ru/ads/system/context.js', () =>
+    renderAdFox(global, data)
   );
 }
 
 /**
  * @param {!Window} global
- * @param {Object} data
+ * @param {!Object} data
  */
-function initAdFox(global, data) {
-  const params = JSON.parse(data.adfoxParams);
+function renderAdFox(global, data) {
+  const renderTo = 'adfox_container';
 
-  createContainer(global, 'adfox_container');
+  createContainer(global, renderTo);
 
   global.Ya.adfoxCode.create({
     ownerId: data.ownerId,
-    containerId: 'adfox_container',
-    params,
-    onLoad: (data, onRender, onError) => {
-      checkLoading(global, data, onRender, onError);
-    },
+    containerId: renderTo,
+    params: JSON.parse(data.adfoxParams),
     onRender: () => global.context.renderStart(),
     onError: () => global.context.noContentAvailable(),
     onStub: () => global.context.noContentAvailable(),
   });
 }
 
-/**
- * @param {!Window} global
- * @param {!Object} data
- * @param {!Object} onRender
- * @param {!Object} onError
- * @return {boolean}
- */
-function checkLoading(global, data, onRender, onError) {
-  if (data.bundleName === 'banner.direct') {
-    const dblParams = {
-      blockId: data.bundleParams.blockId,
-      data: data.bundleParams.data,
-      onRender,
-      onError,
-    };
-
-    yandex(global, dblParams);
```

Comment:
Using yandex is not necessary anymore. `context.js` loader can render both RTB and ADFOX ads.